src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/BuildCRSPresentationLayer.sql

```diff
@@ -0,0 +1,19 @@
+﻿CREATE PROCEDURE [dbo].[BuildCRSPresentationLayer]
+(
+   @RunId int
+)
+AS
+-- ===============================================================================
+-- Author:      Sarma Evani
+-- Create Date: 18/12/2020
+-- Description: Build Vacancies Presentation Layer
+-- ===============================================================================
+BEGIN
+
+		/* Import CRS Data to Presentation Layer */
+
+		EXEC dbo.ImportCRSFrameworkToPL @RunId
+
+		EXEC dbo.ImportCRSSectorStandardToPL @RunId
+
```

Comment:
I see the same Procs in Build Modelled PL. What is this Proc used for ?

Reply:
All FAT V2 modelled functionality is using Build Modelled PL stored procedure